src/pet.js

```diff
@@ -0,0 +1,76 @@
+const minFitness = 0;
+const minAcceptableFitness = 3;
+const maxFitness = 10;
+const minHunger = 0;
+const minAcceptableHunger = 5;
+const maxHunger = 10;
+const maxAge = 30;
+
+function Pet(name) {
+  this.name = name;
+  this.age = 0;
+  this.hunger = minHunger;
+  this.fitness = maxFitness;
+}
+
+Pet.prototype = {
+  get isAlive() {
+    return (
+      this.age < maxAge &&
+      this.hunger < maxHunger &&
+      this.fitness > minFitness
+    );
+  },
+};
+
+Pet.prototype.growUp = function () {
+  if (!this.isAlive) {
+    throw new Error("(*_*) Your pet has died.");
+  }
```

Comment:
you have written the line `throw new Error("(*_*) Your pet has died.");` multiple times throughout this project. You can make this code DRY-er by saving parts of it to a variable, e.g. `const DEAD_PET_ERROR = "(*_*) Your pet has died."`